web/packages/shared/components/UnifiedResources/types.ts

```diff
@@ -100,10 +120,10 @@ export type UnifiedResourcesQueryParams = {
     fieldName: string;
     dir: 'ASC' | 'DESC';
   };
-  pinnedOnly?: boolean;
```

Comment:
I believe that removing this makes the data model more clean. Currently, there is no single source of truth: this property is defied both in the params and in the preferences. Ideally it would be kept only in the preferences.